src/crawlers/playwright_crawler.js

```diff
@@ -245,7 +249,7 @@ class PlaywrightCrawler extends BrowserCrawler {
         browserPoolOptions.browserPlugins = [
             new PlaywrightPlugin(
                 // eslint-disable-next-line
-                getPlaywrightLauncherOrThrow(playwrightModule),
+                getPlaywrightLauncherOrThrow(launcher),
```

Comment:
`getSomething(something)` feels weird. Especially when the only validation in the function is a check if it is an object, which we already check by `ow`. How about removing the parameter from the function and hardcoding `playwright` into the string part of the function?

```js
const {
            launcher = getDefaultPlaywrightLauncher(),
            launchContext = {},
            gotoTimeoutSecs,
            navigationTimeoutSecs,
            browserPoolOptions = {},
            ...browserCrawlerOptions
        } = options;

new PlaywrightPlugin(launcher, ...)
```